tests/pr_test.go

```diff
@@ -71,7 +71,9 @@ func TestRunUpgradeCompleteExample(t *testing.T) {
 		BestRegionYAMLPath: regionSelectionPath,
 		ResourceGroup:      resourceGroup,
 		TerraformVars: map[string]interface{}{
-			"pg_version": "11", // Always lock to the lowest supported Postgres version
+			"pg_version": "11",                                                                                                // Always lock to the lowest supported Postgres version
+			"users":      "[{\n    name = \"testuser\",\n    password = \"password1234\" \n    type     = \"database\"\n  }]", // pragma: allowlist secret
```

Comment:
It parses as a list only when executes in terraform apply.

<img width="1677" alt="image" src="https://github.com/terraform-ibm-modules/terraform-ibm-icd-postgresql/assets/108140212/2fa1c5a6-e3fc-49ea-a95e-734938b35c2e">

Reply:
hmm ok. I guess it can be cleanup though. It doesnt need to newline chars does it?: 
`[{name = \"testuser\", password = \"password1234\", type = \"database\"}]`